test/rules/base.jl

```diff
@@ -1,10 +1,31 @@
-function test_scalar(f, f′, xs...)
+_isapprox(x, y; kwargs...) = isapprox(x, y; kwargs...)
```

Comment:
Why is this necessary?

Reply:
Mainly because `isapprox` isn't implemented for `Zero()` and `One()`. It might make sense to overload `Base.isapprox` instead, but I don't know if that belongs in this PR.

Reply:
If it's relevant to this change then it's fine in this PR. It's better than adding a separate function at any rate.